panacea/query_client.go

```diff
@@ -195,53 +192,40 @@ func newTMLogger(conf *config.Config) tmlog.Logger {
 	return logger
 }
 
-func (q verifiedQueryClient) safeUpdateLightClient(ctx context.Context) (*tmtypes.LightBlock, error) {
+// startSchedulingLastBlockCaching updates the LightClient with the last block information and stores the height of this block.
+func (q *verifiedQueryClient) startSchedulingLastBlockCaching() {
+	for {
+		lastHeight, err := q.GetLastBlockHeight(context.Background())
+
+		if err != nil {
+			log.Errorf("failed to refresh last block. %v", err)
+			return
```

Comment:
What do you think about just logging an error and continue this loop? because this is a goroutine and the `GetLastBlockHeight` can be failed due to some temporary issues on the chain.

Reply:
If so, i think we also need to sleep before continuing the loop.

Reply:
OK. You're right.
I'm going to delete the `return` and let this loop continue even if an error occurs.

Reply:
I fixed it (https://github.com/medibloc/panacea-oracle/pull/101/commits/29271485518fe0daf363103a39f1c7e314f59c37)

Reply:
yeah. i think we don't need to update last height if the error occurs, using `else`.

Reply:
Ah sorry, I now understand exactly what you mean.

Reply:
If the last block information is not available when the server is first started, the server will fail to start. 
Because without the last block information, all queries on this server will fail.
fixed(https://github.com/medibloc/panacea-oracle/pull/101/commits/569ca81e18fc59a858c8f73a0907d53b3574b0e8)

Reply:
:+1: